.travis.yml

```
@@ -27,7 +27,7 @@ install:
 - curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.28.2
 
 script:
-- go test ./... -race -coverprofile=coverage.txt -covermode=atomic
+- go test ./... -coverprofile=coverage.txt -covermode=atomic
```

In this case the `set` is enough as covermode. 
```suggestion
- go test ./... -coverprofile=coverage.txt -covermode=set
```
This coverage calculation is enough for our project purpose, temporarily.